ci-usd/README.md

```diff
@@ -81,6 +83,7 @@ Contains:
 * pyside-5.15.2
 * cmake-3.19.3
 * imath-2.5.5
+* materialx-
```

Comment:
Looks like something may be broken here and it didn't pull ASWF_MATERIALX_VERSION?  Or is it because this version of USD didn't require MaterialX?